sparkpost/base.py

```diff
@@ -22,6 +22,8 @@ def request(self, method, uri, **kwargs):
         response = requests.request(method, uri, headers=headers, **kwargs)
         if not response.ok:
             raise SparkPostAPIException(response)
+        if response.status_code == 204:
```

Comment:
I think we need to make some kind of change to the base Resource. We'll have to discuss this.